doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -221,21 +221,25 @@ as simple as::
 Automated PR validation
 -----------------------
 
-When your PR is created or updated, the Ceph project's `Continuous Integration
+When you create or update your PR, the Ceph project's `Continuous Integration
 (CI) <https://en.wikipedia.org/wiki/Continuous_integration>`_ infrastructure
-will test it automatically. At the time of this writing (September 2020), the
-automated CI testing included five tests to check that the commits in the PR
-are properly signed (see :ref:`submitting-patches`), to check that the
-documentation builds, to check that the submodules are unmodified, to check
-that the API is in order, and a :ref:`make-check` test.  Additional tests may
-be performed depending on which files are modified by your PR.
-
-The :ref:`make-check`, builds the PR and runs it through a battery of
+automatically tests it. At the time of this writing (September 2020), the
+automated CI testing included five tests: 
+
+#. a test to check that the commits are properly signed (see :ref:`submitting-patches`): 
+#. a test to check that the documentation builds
+#. a test to check that the submodules are unmodified
+#. a test to check that the API is in order
+#. a make-check test  
```

Comment:
should we update this list?

Reply:
"make-check" was a hyperlink before this change . . . could we preserve that?

Reply:
Sure thing, Nathan. I'll put it back, and I'll do so in a way that improves the doc.

Here's why I took it out in the first place: by default, the way that Sphinx creates hyperlinks involves inserting the text of the section that the link targets. This means that the sentence read "The What does "make check" mean test, builds", which is gibberish. Fortunately for all of us, I got the text to look the way I wanted and also preserved the hyperlink to the make-check section of the docs. So have another look and let me know if it looks right to you.